doc/tutorials/11-ferrofluid/11-ferrofluid_part2.ipynb

```diff
@@ -66,10 +66,10 @@
     "and set up the simulation parameters where we introduce a new dimensionless parameter \n",
```

Comment:
<p>we should use formatting strings:</p><p> </p><p> <code>print(f"value of i: {i}")</code></p><p> </p><p>that's much more readable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>